src/commands/generateEditorConfig.ts

```diff
@@ -0,0 +1,45 @@
+import {exists, writeFile} from 'fs';
+import {workspace, window} from 'vscode';
+import {Utils} from '../utils';
+
+const supportedProperties = ['indent_style', 'indent_size', 'tab_width'];
```

Comment:
This is for generating a `.editorconfig` file. So what values should be used then for those properties. We can't read them from the user settings. Except for `trim_trailing_whitespace`. That one can be added.